generators/readme/package.json

```diff
@@ -33,6 +33,7 @@
     "remark-stringify": "9.0.1",
     "shelljs": "0.8.4",
     "to-vfile": "6.1.0",
+    "typescript-react-function-component-props-handler": "1.1.1",
```

Comment:
This dependency is used to "fix" an issue with `react-docgen` when using `FC<>`. See https://github.com/reactjs/react-docgen/issues/387